src/pages/computer-science/what-is-an-api/index.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: What Is an API
+title: What Is an Api
```

Comment:
Can you undo the changes in this file?

Reply:
This is not a change I made. I found the reason to be the pre-commit normalization. I changed it to API. After committing, it is back to API and the file is unstaged. Should this be opened as an issue?

Reply:
@systimotic I found the reason to be the titleify function. Since 'API' is not in preformatted words list, it is making it 'Api'.

Reply:
@dhanushuUzumaki Yep, that's correct, good found! Can you add it to the `preformatted` array?